src/partials/modals/franchise.html

```diff
@@ -0,0 +1,39 @@
+<!-- <div class="backdrop is-hidden" data-modal>
+  <div class="franchise__modal">
+    <button type="button" class="franchise__button">
+      <svg class="location__icon" width="24" height="24">
+        <use href="./images/icons.svg#mobile_menu_x"></use>
+      </svg>
+    </button>
+    <h2 class="franchise__title">Thank you for your interest in owning an Ice Cream shop</h2>
+    <p class="franchise__text">
+      Due to the overwhelming volume of franchise requests we are not accepting new franchisee
+      applications at this time. Once the application process resumes, we will contact you for
+      further information. Please fill out the contact information below and we will reach out for
+      additional information. We appreciate your patience.
+    </p>
+
+    <form class="form">
+      <div class="form__wrap">
+        <label for="name"></label>
+        <input class="form__input" name="name" id="name" placeholder="Name" />
+
+        <label for="tel"></label>
+        <input class="form__input" name="tel" id="tel" placeholder="Phone" />
```

Comment:
Should be added pattern for phone number validation + required.

Reply:
add